airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/util/ConnectorExceptionHandler.kt

```diff
@@ -145,7 +145,7 @@ open class ConnectorExceptionHandler {
     open fun translateConnectorSpecificErrorMessage(e: Throwable?): String? {
         if (e == null) return null
         for (error in connectorErrorDictionary) {
-            if (e.message?.lowercase()?.matches(error.regexMatchingPattern.toRegex())!!)
+            if (e.message?.lowercase()?.matches(error.regexMatchingPattern.lowercase().toRegex())!!)
```

Comment:
rather than doing that, I'd rather store a pattern in the error object, and compile it with compilePattern(CASE_INSENSITIVE)

Reply:
good idea! fixed, please take another look. thanks!